deploy/helm/CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## [2.5.0-alpha.7] - 2023-06-08
```

Comment:
please call this `2.6.0-alpha.1`, this is too dangerous to ship this as last minute together with 2.5.0